android/java/res/layout/item_custom_filter.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) 2023 The Brave Authors. All rights reserved.
+     This Source Code Form is subject to the terms of the Mozilla Public
+     License, v. 2.0. If a copy of the MPL was not distributed with this file,
+     You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:app="http://schemas.android.com/apk/res-auto" 
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <TextView
+          android:id="@+id/title_text"
+          android:layout_width="0dp"
+          android:layout_height="wrap_content"
+          app:layout_constraintTop_toTopOf="parent"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintEnd_toStartOf="@id/toggle_switch"
+          android:maxLines="1"
+          android:layout_marginTop="10dp"
+          android:layout_marginHorizontal="16dp"
+          android:textSize="16sp"
+          android:textColor="@color/filter_title_color"/>
+
+     <TextView
+          android:id="@+id/last_update_text"
+          android:layout_width="0dp"
+          android:layout_height="wrap_content"
+          app:layout_constraintTop_toBottomOf="@id/title_text"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintEnd_toStartOf="@id/toggle_switch"
+          android:layout_marginTop="2dp"
+          android:layout_marginHorizontal="16dp"
+          android:textSize="15sp"
```

Comment:
we should try to keep sizes with multiple of 4's if possible.

Reply:
I have changed it to 14sp. If I keep 16sp then it will be same as title and 12sp will be too small.